apps/web/components/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -171,11 +181,17 @@ function EventTypeSingleLayout({
     t("locked_fields_member_description")
   );
 
+  const enabledWebhooks = formMethods.watch("enabledWebhooks");
+  const length = formMethods.watch("length");
+  const multipleDuration = formMethods.watch("metadata")?.multipleDuration;
+
```

Comment:
Instead of maintaining `enabledWebhooks` separately, we can simply invalidate eventType tRPC query like this [here](https://github.com/calcom/cal.com/blob/f87eac193fb0a6b244489a61b97e5f65b73e64b9/packages/features/webhooks/components/WebhookListItem.tsx#L53) 

```javascript
if (webhook.eventTypeId) {
    await utils.viewer.eventTypes.get.invalidate({ id: webhook.eventTypeId });
  }
```